package/base-files/files/etc/rc.common

```diff
@@ -62,20 +62,36 @@ depends() {
 	return 0
 }
 
+EXTRA_HELP=""
+EXTRA_COMMANDS=""
+extra_command() {
+	local cmd="$1"
+	local help="$2"
+
+	local extra
+
+	extra="$(printf "%-16s%s" "${cmd}" "${help}")"
+
+	EXTRA_HELP="\t${extra}\n${EXTRA_HELP}"
+	EXTRA_COMMANDS="${EXTRA_COMMANDS} ${cmd}"
+}
+
 help() {
 	cat <<EOF
 Syntax: $initscript [command]
 
 Available commands:
-	start	Start the service
-	stop	Stop the service
-	restart	Restart the service
-	reload	Reload configuration files (or restart if service does not implement reload)
-	enable	Enable service autostart
-	disable	Disable service autostart
-	enabled	Check if service is started on boot
-$EXTRA_HELP
 EOF
+
+	extra_command "enabled" "Check if service is started on boot"
+	extra_command "disable" "Disable service autostart"
+	extra_command "enable" "Enable service autostart"
+	extra_command "reload" "Reload configuration files (or restart if service does not implement reload)"
+	extra_command "restart" "Restart the service"
+	extra_command "stop" "Stop the service"
+	extra_command "start" "Start the service"
```

Comment:
When you call `help()` twice (for whatever reason), you will get different output - the commands will be duplicated.

Reply:
ALL_COMMANDS variable at the end of `rc.common` also uses EXTRA_COMMANDS, so those commands (`enabled`, `disable`, `enable`, `reload`, `restart`, `stop`, `start`) will be doubled.

Reply:
@oldium This should be fixed with my last change